components/compliance-service/integration_test/reporting_server_list_suggestions_test.go

```diff
@@ -0,0 +1,206 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	iam_v2 "github.com/chef/automate/api/interservice/authz/v2"
+	authzConstants "github.com/chef/automate/components/authz-service/constants/v2"
+	"github.com/chef/automate/components/compliance-service/api/reporting"
+	reportingServer "github.com/chef/automate/components/compliance-service/api/reporting/server"
+	"github.com/chef/automate/components/compliance-service/ingest/events/compliance"
+	"github.com/chef/automate/components/compliance-service/reporting/relaxting"
+	rules_tags "github.com/chef/automate/lib/authz"
+)
+
+func ingestReport(fileName string, f func(*compliance.Report)) error {
+	fileData, err := os.Open(fileName)
+	if err != nil {
+		return err
+	}
+	defer fileData.Close()
+	var iReport compliance.Report
+	unmarshaler := &jsonpb.Unmarshaler{AllowUnknownFields: true}
+	if err = unmarshaler.Unmarshal(fileData, &iReport); err != nil {
+		return err
+	}
+
+	f(&iReport)
+
+	ctx := context.Background()
+	_, err = suite.ComplianceIngestServer.ProcessComplianceReport(ctx, &iReport)
+
+	return err
+}
```

Comment:
This function takes the filename of one of our json profiles on disk, reads it, and sends it through the normal ingestion pipeline. This was done so that these tests could be somewhat unaware of where the data was coming from as far as summary index vs report index in elastic.